### PR TITLE
Tests for different S3 methods

### PR DESCRIPTION
**Pull request by RomanTsegelskyi**
More test cases for CrossTable (aslo one small fix), coxph, clogit, ts

### CODE REVIEW

**Comment by RomanTsegelskyi** [comment]
@daroczig, I think I am done with tests for different S3 methods. Should I just merge it?

File: R/S3.R [hunk 1]
```diff
@@ -1077,151 +1085,75 @@ pander.zoo <- function(x, caption = attr(x, 'caption'), ...) {
...
+            'Observations'        = xs$dims[["N"]],
+            'Groups'              = xs$dims$ngrps[1:xs$dims$Q],
+            'Log-restricted-likelihood' = xs$logLik,
             check.names = FALSE), keep.trailing.zeros = TRUE, caption = caption, digits = 4)
     } else {
-
+        
         pandoc.table(res, caption = caption, ...)
-
+
```
Comment (daroczig) [inline R/S3.R]: Can you please also remove the white space from the empty lines? My future commits would automatically revert this, so we might end up adding and removing white space all the time.

**Comment by daroczig** [comment]
Otherwise it looks OK and extremely useful, thank you. After fixing the above mentioned small code style issues, can you please merge?

**Comment by RomanTsegelskyi** [comment]
@daroczig I fixed the style issues and after some more thinking removed exclusions

**Comment by daroczig** [comment]
Thanks!

**Pull request merged**